src/Components/Book.js

```diff
@@ -10,23 +10,51 @@ const Book = (props) => {
   const dispatch = useDispatch();
 
   return (
-    <article>
-      <h2>{title}</h2>
-      <h3>{author}</h3>
-      <p>{category}</p>
-      <ul className={styles.book_btn_container}>
-        <li>
-          <button type="button">Comments</button>
-        </li>
-        <li>
-          <button type="button" onClick={() => dispatch(deleteBook(id))}>
-            Remove
-          </button>
-        </li>
-        <li>
-          <button type="button">Edit</button>
-        </li>
-      </ul>
+    <article className={styles.article}>
```

Comment:
- [ ] After making the changes suggested,  Some elements appear to be positioned wrongly as there is supposed to be a section on the left containing the book details and one on the right containing the progress. Kindly check that out to match the original design

![image](https://github.com/bilalrajput09/Bookstore-react/assets/98453979/7fef7b8d-e434-4181-bddf-acca2587d186)

Reply:
Fixed!!!